src/features/user/MFV2/ContributionsMap/Popup/ProjectInfoSection.tsx

```diff
@@ -0,0 +1,76 @@
+import { useTranslations } from 'next-intl';
+import style from '../MyForestV2.module.scss';
+import { CountryCode } from '@planet-sdk/common';
+import DonateButton from '../../MyContributions/DonateButton';
+import { PointFeature } from 'supercluster';
+import {
+  ProfilePageType,
+  DonationProperties,
+} from '../../../../common/types/myForestv2';
+
+interface ProjectInfoSectionProps {
+  superclusterResponse: PointFeature<DonationProperties>;
+  profilePageType: ProfilePageType;
+  supportedTreecounter?: string;
+}
+const ProjectInfoSection = ({
+  superclusterResponse,
+  profilePageType,
+  supportedTreecounter,
+}: ProjectInfoSectionProps) => {
+  const { totalContributionUnits, contributionUnitType } =
+    superclusterResponse.properties.contributionInfo;
+  const { tpoName, country, purpose, slug, unitType } =
+    superclusterResponse.properties.projectInfo;
+  const tProfile = useTranslations('Profile');
+  const tCountry = useTranslations('Country');
+  const _country: Lowercase<CountryCode> =
+    country.toLocaleLowerCase() as Lowercase<CountryCode>;
+
+  const totalContributedUnits =
+    contributionUnitType === 'tree'
+      ? tProfile('myForestMap.plantedTree', {
+          count: Number.isInteger(totalContributionUnits)
+            ? totalContributionUnits
+            : totalContributionUnits.toFixed(2),
+        })
+      : tProfile('myForestMap.savedArea', {
+          area: Number.isInteger(totalContributionUnits)
+            ? totalContributionUnits
+            : totalContributionUnits.toFixed(2),
+        });
```

Comment:
We need to consider both restored and conserved area.

e.g. 
 - 150 m² conserved (if project purpose is conservation)
 - 150m² restored (if project purpose is trees and unitType is m2)

Reply:
@mariahosfeld said that we don't need to show we just need to show `m2`